app/services/current_team_and_user/ico/standard.rb

```diff
@@ -1,6 +1,6 @@
 module CurrentTeamAndUser
   module ICO
-    class FOI < ::CurrentTeamAndUser::Base
+    class Standard < ::CurrentTeamAndUser::Base
```

Comment:
It's because [CurrentTeamAndUser#resolver_for_case](https://github.com/ministryofjustice/correspondence_tool_staff/blob/b7ce1a791fdfa83fea6ae27ba2757fa4c1a8ccdd/app/services/current_team_and_user_service.rb#L23) uses the workflow to determine the subclass to instantiate, presumably because the current team and user depends on that, not on the case sub-class.